### ISSUE TITLE
[BUG] Playlist button in current playing playlist error

### ISSUE THREAD

**Issue by Gorge-1** [jurialmunkey/skin.arctic.horizon.2#901]
### Skin section

Music Visualisation OSD

### Current Behavior

When in the Music Visualisation OSD, we can open the current playing playlist by pressing DOWN. Next to the playlist is a button "Playlist". Clicking this button gives an error "Playback failed".

### Expected Behavior

I don't know what this button should do.

### Steps To Reproduce

Go to songs.
Play a song.
Open the OSD.
Press DOWN.
Press Playlist button.

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Are you talking about the listitem with the headphones on a yellow background?

It plays the folder `playlistmusic://` which is the currently queued music playlist. It works fine for me playing music in my library.

**Comment by Gorge-1** [comment]
Yes thats the button I mean, I get the error after a couple of songs have played.

**Comment by jurialmunkey** [comment]
Debug log is needed.

**Comment by ceefaxer** [comment]
Did you get a debug log for this or do you need one? I have the same issue. 
When on the OSD you have the headphone icon highlighted and you see the album or playlist on the right. When you navigate to select a different track and select it crashes back to main library with error message 'One or more items failed to play. Check log for more information about this message.'
